src/Scientist/Internals/ExperimentInstance.cs

```diff
@@ -64,6 +66,11 @@ public async Task<T> Run()
                 return await Behaviors[0].Behavior();
             }
 
+            if (!await ShouldControlRun())
+            {
+                return await Behaviors[1].Behavior();
```

Comment:
This does not allow having multiple candidates, which makes sense in the context of the PR, but maybe this can be called out stronger by changing the name of `ShouldControlRun` to something like `ReturnFirstCandidate`?

Reply:
Apologizes for the delay, 
I agree, i'll make the change.

Reply:
renamed the property as requested